bin/resources/GameIndex.yaml

```diff
@@ -14729,8 +14729,8 @@ SLES-52782:
   gsHWFixes:
     cpuSpriteRenderBW: 1 # Fixes textures.
 SLES-52783:
-  name: "Call of Duty - Le Jour de Gloire"
-  region: "PAL-F"
+  name: "Call of Duty - Le Jour de Gloire-Finest Hour-L'Ora Degli Eroi"
```

Comment:
It doesn't have half of this on the box. https://psxdatacenter.com/psx2/images2/covers/SLES-52783/SLES-52783-F-S.html

Reply:
Because it's the link of the french cover.
SLES-52783 is the same serial for :
- french `Call of Duty : Le Jour de Gloire`
- spanish (non localized title) `Call of Duty: Finest Hour` => https://i.ebayimg.com/images/g/osUAAOSw5HNimp2Z/s-l500.png
- italian => https://m.media-amazon.com/images/I/51ZruC1fRlL._AC_.jpg

So I've regrouped every localized title.

Reply:
Okay there's no need to write every title under the sun there. just "Finest Hour" is fine.

Reply:
OK !
I'll revert the title of SLES-52046 to `007 - Everything or Nothing`

Reply:
if it's just a single foreign language title, it's fine, but if it's multiple different ones for the same serial, then just use a single title, whichever one, I don't really care :P English is usually preferred in this scenario though.

Reply:
Really need additionnal lines in GameIndex so we can add every localized title for a single serial. /o/

Reply:
But which one do you display? there's no distinction on the disc between regions :P

Reply:
Could be an option chosen by the user through the interface of PCSX2, something like "prefered game title language".

Reply:
Eh, sounds a bit excessive for something most people don't see because they have it full screen.

Reply:
I'm just a database nerd (and that's part of my job at the french branch of Geeks-Line), so the more options you give me, the more happy I am, you know :D

Reply:
I appreciate the effort :)